src/invoke-prompt-fail-client.ts

```diff
@@ -0,0 +1,34 @@
+import { Connection, Client } from '@temporalio/client';
+import { invokePromptWorkflow } from './workflows';
+import { nanoid } from 'nanoid';
+
+async function run() {
+  const connection = await Connection.connect({ address: 'localhost:7233' });
+
+  const client = new Client({
+    connection
+  });
+
+  const [ latestDocumentProcessingId, query, conversationId ] = process.argv.slice(2)
+
+  const id = `invoke-prompt-workflow-fail-${nanoid()}`.toLowerCase().replaceAll('_', '')
+  const handle = await client.workflow.start(invokePromptWorkflow, {
+    taskQueue: 'invoke-prompt-queue',
+    args: [{
+      query,
+      latestDocumentProcessingId,
+      conversationId,
+      tossError: true
```

Comment:
Can this be passed as an argument?

Reply:
I believe the entire object here is the arg input; which is the advice of Temporal when passing args; usually pass an arg object over individual args

Reply:
I meant passed as an argument to the script, so it could be set on the command line.